device-api/src/blocking.rs

```diff
@@ -93,10 +90,14 @@ fn list_devfs<P: AsRef<Path>>(devfs: P) -> io::Result<Vec<DevFile>> {
 }
 
 fn is_furiosa_device(idx: u8, sysfs: &str) -> bool {
-    std::fs::read_to_string(npu_mgmt::path(sysfs, PLATFORM_TYPE, idx))
-        .ok()
-        .filter(|c| npu_mgmt::is_furiosa_platform(c))
-        .is_some()
+    std::fs::read_to_string(npu_mgmt::path(
+        sysfs,
+        npu_mgmt::StaticMgmtFile::PlatformType.path(),
+        idx,
+    ))
```

Comment:
I have a minor opinion. Which do you think is better?

```
npu_mgmt::path(sysfs, npu_mgmt::StaticMgmtFile::PlatformType.path(), idx)
```
or
```
npu_mgmt::StaticMgmtFile::PlatformType.path(sysfs, idx)
```


I'm not sure if it's proper for the `fn path()` in `trait MgmtFile` to return a `&str` rather than a `PathBuf`.

Reply:
I agree with that `fn path()` returns `&str` is not proper. Then, how about rename `fn path()` to `fn filename()`? There are some usage of current `fn path()` such as [here](https://github.com/furiosa-ai/device-api/pull/83#pullrequestreview-1517636358). And I think `fn filename()` returns '&str' is proper like [std::Path](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.file_name).

Reply:
I think `filename` is better.

Reply:
Reflect at [6f3bc5e](https://github.com/furiosa-ai/device-api/pull/83/commits/6f3bc5ece0628d32a1ba7d2ad12b330cd951c191)